### DISCUSSION TITLE
How to do batch inference

### DISCUSSION THREAD

**Discussion by peixin-lin** [PaddlePaddle/PaddleOCR#13611] [Q&A]
When I perform OCR on a large PDF file that contains many pages (like 200+), the whole procedure would take a very long time. I am wondering if there is a way to do batch inference using PPStucture or with a lower level function? Please advise. Thanks!

**Comment by UserWangZz** [comment]
Currently, the OCR-deteciton model does not support batch reasoning. The only conceivable method is multi-threaded calling, which may require you to write the corresponding code yourself.
